components/swatch/stories/template.js

```diff
@@ -146,70 +132,37 @@ export const Template = ({
 
 /* Shows a single group of swatches with all rounding options. */
 export const RoundingGroup = (args, context) => Container({
-	withBorder: false,
 	content: html`
 		${Container({
-			withBorder: false,
 			heading: "Regular (Default)",
 			containerStyles: { "gap": "8px" },
 			content: Template(args, context),
 		}, context)}
 		${Container({
-			withBorder: false,
 			heading: "Full",
 			containerStyles: { "gap": "8px" },
 			content: Template({...args, rounding: "full", }, context),
 		}, context)}
 		${Container({
-			withBorder: false,
 			heading: "None",
 			containerStyles: { "gap": "8px" },
 			content: Template({...args, rounding: "none", }, context),
 		}, context)}
 	`
 }, context);
 
-/* Shows a single group of swatches with all border options. */
-export const BorderGroup = (args, context) => Container({
-	withBorder: false,
-	content: html`
-		${Container({
-			withBorder: false,
-			heading: "Default",
-			containerStyles: { "gap": "8px" },
-			content: Template(args, context),
-		}, context)}
-		${Container({
-			withBorder: false,
-			heading: "No border",
-			containerStyles: { "gap": "8px" },
-			content: Template({...args, borderStyle: "noBorder"}, context),
-		}, context)}
-		${Container({
-			withBorder: false,
-			heading: "Light Border",
-			containerStyles: { "gap": "8px" },
-			content: Template({...args, borderStyle: "lightBorder"}, context),
-		}, context)}
-	`
-}, context);
-
 /* Shows a single group of swatches that are empty/nothing in various shapes and rounding. */
 export const EmptyGroup = (args, context) => Container({
-	withBorder: false,
 	content: html`
 		${Container({
-			withBorder: false,
```

Comment:
We can restore these also as they're on the Container!

![image](https://github.com/user-attachments/assets/9f12f5a9-60d9-45e4-a1b5-b20d5cff4010)